### ISSUE TITLE
[scss] syntax does not support #{&}

### ISSUE THREAD

**Issue by MRokas** [Microsoft/vscode#10361]
- VSCode Version: 1.4.0
- OS Version: Ubuntu 14

Steps to Reproduce:

1. Open Scss/Sass file, or choose that language
2. Use `@at-root`, example:
```
.head {
  height: 64px;
  line-height: 64px;
  background-color: #f2f2f2;
  margin-top: $gradientHeight;

  @at-root #{&}__title {
    font-size: 2rem;
    color: #339;
    margin-left: 2rem;
    display: inline-block;
  }
```

[Screenshot example](http://i.imgur.com/wboc6Mp.png)

It has been reported earlier too - #4524

**Comment by aeschli** [comment]
Please try https://code.visualstudio.com/insiders.
@at-root is now supported, but #{&} isn't accepted yet. I'd suggest to change the title of the issue to that.

**Comment by MRokas** [comment]
@aeschli - thanks for the tip, I changed it!

**Comment by aeschli** [comment]
fixed by vscode-css-languageservice@0844f07c25a8dc6e4a1a83ca3bff0d613b65e4f5
